solvation_analysis/analysis.py

```diff
@@ -0,0 +1,103 @@
+# from abc import ABC
+
+
+from MDAnalysis.analysis.base import AnalysisBase
+from MDAnalysis.analysis.rdf import InterRDF
+import numpy as np
+
+
+class Solute:
+    def __init__(self, solute, solvents, **kwargs):
+        super(Solute, self).__init__(solute.universe.trajectory, **kwargs)
+        self.solute = solute
+        self.solvents = solvents
+
+    def run(self):
+        # run all RDFs and store their results
+        rdfs = []
+        for solvent in self.solvents:
+            rdf = InterRDF(self.solute, solvent)
+            rdf.run()
+            rdfs.append(rdf)
+        self.rdfs = rdfs
+        self.ion_speciation = IonSpeciation(self.solute, self.solvents)
```

Comment:
the user will need to call run on these sub methods in current design. eg:
``` my_solute.ion_speciation.run()```
which I kind of like, however I would move the class construction to `_conclude()`